### DISCUSSION TITLE
Option to ignore hitsound volume changes

### DISCUSSION THREAD

**Discussion by crumpledmint** [ppy/osu#21136] [Ideas] [answered]
It would be nice if there was an option to ignore hitsound volume changes since on some maps it's really hard to follow the rhythm when you can't hear hitsounds. It feels like it is as necessary as option to customize background dim

**Comment by Doksha** [comment, marked as answer]
this has already been requested (I requested this too without realizing it was requested before)
[18654](https://github.com/ppy/osu/discussions/18654)
and
[20784](https://github.com/ppy/osu/discussions/20784)

**Reply by crumpledmint** [reply]
Yeah, my bad
